src/prop/theory_proxy.cpp

```diff
@@ -98,7 +98,8 @@ void TheoryProxy::explainPropagation(SatLiteral l, SatClause& explanation) {
 
   TrustNode tte = d_theoryEngine->getExplanation(lNode);
   Node theoryExplanation = tte.getNode();
-  if (d_env.isTheoryProofProducing())
+  if (options::produceProofs()
```

Comment:
I think this is the same as d_env.isSatProofProducing right?

Reply:
Indeed. I'll change.